arrows/core/class_probablity_filter.h

```diff
@@ -63,18 +63,17 @@ class KWIVER_ALGO_CORE_EXPORT class_probablity_filter
   : public vital::algorithm_impl<class_probablity_filter, vital::algo::detected_object_filter>
 {
 public:
-  static constexpr char const* name = "class_probablity_filter";
-
-  static constexpr char const* description =
-    "Filters detections based on class probability.\n\n"
-    "This algorithm filters out items that are less than the threshold."
-    " The following steps are applied to each input detected object set.\n\n"
-    "1) Select all class names with scores greater than threshold.\n\n"
-    "2) Create a new detected_object_type object with all selected class"
-    " names from step 1. The class name can be selected individually"
-    " or with the keep_all_classes option.\n\n"
-    "3) The input detection_set is cloned and the detected_object_type"
-    " from step 2 is attached.";
+  PLUGIN_INFO( "class_probablity_filter",
```

Comment:
To keep line lengths shorter, we may want to use this style:

```c++
public:
  PLUGIN_INFO( "item_name", "1.0",
    "This is the first line of the description text "
    "and it may continue on multiple lines"
  )
```